### PR TITLE
Stats: add upgrade copy for the purchase page

### PR DESCRIPTION
**Pull request by kangzj**
Related to #85117

## Proposed Changes

Changes copy for tier upgrades, i.e. when user has a plan for the site already. I intentionally tweaked the copy a bit with GPT, so that they are more accurate.

## Testing Instructions

* Apply the latest Diff D129634-code and sandbox `public-api.wordpress.com` and `Store Sandbox`.
* Spin this change up on `local` Calypso.
* Purchase a plan for a site `/stats/purchase/:siteSlug?productType=commercial&flags=stats/tier-upgrade-slider`

image

* Open the purchase page again
* Ensure the copy is like the following

image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx [hunk 1]
```diff
@@ -132,8 +152,10 @@ Thanks\n\n`;
 	return (
 		<>
 			<h1>{ translate( 'Jetpack Stats' ) }</h1>
-			<p>{ translate( 'The most advanced stats Jetpack has to offer.' ) }</p>
-			<StatsBenefitsCommercial />
+			{ ! isCommercialOwned && (
```
Comment (dognose24) [inline client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx:155]: Would it be clearer to consolidate the `! isCommercialOwned` check for only once?
```suggestion
			{ ! isCommercialOwned && (
				<>
					<p>{ translate( 'The most advanced stats Jetpack has to offer.' ) }</p>
					<StatsBenefitsCommercial />
				</>
			) }
			{ isCommercialOwned && <StatsUpgradeInstructions /> }
```

**Review by dognose24 — APPROVED** [review]
LGTM! 👍🏼

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/10643785

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @kangzj for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/10643785).

**Pull request merged**